### PR TITLE
NAS-116358 / 24.10 / Refactor Disks page to ix-tables

### PR DESCRIPTION
**Pull request by bvasilenko**
**Testing**

Go to **Storage > Disks** and test disk filtering, selection and individual actions and batch actions.

### CODE REVIEW

**Review by RehanY147 — CHANGES_REQUESTED** [review]
Pagination missing on the table

File: src/assets/i18n/es.json [hunk 1]
```diff
@@ -4402,7 +4403,6 @@
   "Download": "Descargar",
   "EXPIRED": "EXPIRADO",
   "Edit": "Editar",
-  "Edit Disk(s)": "Editar Disco(s)",
```
Comment (denysbutenko) [inline src/assets/i18n/es.json:4405]: To be aware of existing translations being lost.

**Review by RehanY147 — APPROVED** [review]
Looks good

**Comment by AlexKarpov98** [comment]
@bvasilenko - I noticed that you missed .toLowerCase() when filtering disks, I gonna fix it here: https://github.com/truenas/webui/pull/9566

**Pull request merged**